Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift

```diff
@@ -44,6 +44,10 @@ class SubmissionDetailsViewController: UIViewController, SubmissionDetailsViewPr
         return controller
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
FYI you don't have to call this as of iOS 9+ with `addObserver(_:selector:name:object:)`